core/src/main/scala/com/karumi/shot/reports/ConsoleReporter.scala

```diff
@@ -0,0 +1,65 @@
+package com.karumi.shot.reports
+
+import com.karumi.shot.base64.Base64Encoder
+import com.karumi.shot.domain._
+import com.karumi.shot.ui.Console
+
+class ConsoleReporter(console: Console) {
+
+  def showBase64Error(comparision: ScreenshotsComparisionResult,
+                      outputFolder: String): Unit = {
+    console.show(
+      "🤖  The option printBase64 is enabled. In order to see the generated diff images, run the following commands in your terminal:")
+    console.lineBreak()
+    comparision.screenshots.foreach(screenshot => {
```

Comment:
We should just show the base 64 if the error is ``DifferentScreenshots``. If the image sizes are different or the original screenshot couldn't be found this shouldn't be needed. Did you take this into account?

Reply:
Lol that's true! I'm going to change